### PR TITLE
Update building-a-contact-form.md

### PR DESCRIPTION
**Pull request by mertcanyucel**
Added Getform as a contact form handling solutions

### CODE REVIEW

File: docs/docs/building-a-contact-form.md [hunk 1]
```diff
@@ -43,6 +43,32 @@ Each method detailed below will start with the following contact form:
...
+  </label>
+  <label>
+    Message
+    <input type="text" name="message" />
+  </label>
+  ...
+</form>
+```
+
+Once you've made the changes, you can head over to your contact page and then start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. You can find more info on the registration process and form setup [on their website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).
```
Comment (muescha) [inline docs/docs/building-a-contact-form.md]:
```suggestion
Once you've made the changes, you can head over to your contact page and then start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. You can find more info on the registration process and form setup [on the Getform website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).
```

File: docs/docs/building-a-contact-form.md [hunk 2]
```diff
@@ -43,6 +43,32 @@ Each method detailed below will start with the following contact form:
 
 ## Form submission options in Gatsby
 
+### Getform
+
+Getform is a form backend platform which offers also a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform directing the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```
Comment (laurieontech) [inline docs/docs/building-a-contact-form.md]:
```suggestion
Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform directing the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```

File: docs/docs/building-a-contact-form.md [hunk 3]
```diff
@@ -43,6 +43,32 @@ Each method detailed below will start with the following contact form:
...
+  </label>
+  <label>
+    Message
+    <input type="text" name="message" />
+  </label>
+  ...
+</form>
+```
+
+Once you've made the changes, you can head over to your contact page and then start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. You can find more info on the registration process and form setup on the [Getform website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).
```
Comment (laurieontech) [inline docs/docs/building-a-contact-form.md]:
```suggestion
Once you've made the changes, you can head over to your contact page and start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. 

You can find more info on the registration process and form setup on the [Getform website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).
```

Comment (laurieontech) [reply docs/docs/building-a-contact-form.md]: "you can head over to your contact page and start receiving submissions to the form" can you clarify this a bit? Is there a contact page now in getform? Or is it still in their gatsby site and you're prompting them to send information through the form first?

**Comment by laurieontech** [comment]
Thanks for the PR! This looks like a great addition. Added some comments for review.

File: docs/docs/building-a-contact-form.md (hunk 3 above)
Comment (mertcanyucel) [reply docs/docs/building-a-contact-form.md]: 2nd one it is! It's still in the gatsby site and you're prompting them to send information through the form.

Comment (laurieontech) [reply docs/docs/building-a-contact-form.md]: Cool, can we clarify that in the text?

**Comment by mertcanyucel** [comment]
Could you please merge this PR if all is good guys? @muescha and @laurieontech

**Comment by muescha** [comment]
can you please update your pull requests with the suggestions fromm the review?

**Comment by mertcanyucel** [comment]
@muescha here is the updated PR, hope I made it correctly.  https://github.com/gatsbyjs/gatsby/pull/19570/commits/17b6a1fd8d5db238c5c4ba8dab1ca7da7e1f9ca

**Comment by muescha** [comment]
we should ask @laurieontech for review

**Comment by mertcanyucel** [comment]
@laurieontech could you please take a look at this? thanks

File: docs/docs/building-a-contact-form.md [hunk 4]
```diff
@@ -43,6 +43,31 @@ Each method detailed below will start with the following contact form:
...
+    <input type="text" name="name" />
+  </label>
+  <label>
+    Message
+    <input type="text" name="message" />
+  </label>
+  ...
+</form>
+```
+Once you've made the changes, you can head over to your contact page and start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered.
```
Comment (laurieontech) [inline docs/docs/building-a-contact-form.md]: We still need to be clearer here as to where you're making the submissions and where you can see them.

File: docs/docs/building-a-contact-form.md [hunk 5]
```diff
@@ -43,6 +43,31 @@ Each method detailed below will start with the following contact form:
 
 ## Form submission options in Gatsby
 
+### Getform
+
+Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform directing the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```
Comment (laurieontech) [inline docs/docs/building-a-contact-form.md]:
```suggestion
Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform. When creating the form, direct the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```
Which platform? Gatsby or Getform?

Comment (mertcanyucel) [reply docs/docs/building-a-contact-form.md]: Getform is the platform which can be used as a form submission handling option for Gatsby.

Comment (laurieontech) [reply docs/docs/building-a-contact-form.md]: Right. Just want to add some language to clarify this piece `you must first create a form on the platform`. It isn't clear which platform is being discussed.

File: docs/docs/building-a-contact-form.md [hunk 6]
```diff
@@ -43,6 +43,32 @@ Each method detailed below will start with the following contact form:
...
+    <input type="text" name="name" />
+  </label>
+  <label>
+    Message
+    <input type="text" name="message" />
+  </label>
+  ...
+</form>
+```
+Once you've made the changes on your Gatsby contact form, you can head over to your contact page and start receiving submissions to the form, the submissions will then be visible on Getform dashboard. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data you see on Getform using Zapier and Webhooks options that are offered.
```
Comment (laurieontech) [inline docs/docs/building-a-contact-form.md]:
```suggestion
Once you've made the code changes in your contact form, you can head over to the contact page on your site and start submitting data to the form. The submissions will then be visible on the Getform dashboard. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data you see on Getform using Zapier and Webhooks options that are offered. 
```
There are a few different levels here. The form code in Gatsby, the form itself in your site, and the Getform dashboard. I made some recommendations that I hope will clarify which piece we're referring to.

File: docs/docs/building-a-contact-form.md [hunk 7]
```diff
@@ -43,6 +43,32 @@ Each method detailed below will start with the following contact form:
 
 ## Form submission options in Gatsby
 
+### Getform
+
+Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the Gatsby site. When creating the form, direct the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```
Comment (laurieontech) [inline docs/docs/building-a-contact-form.md]:
```suggestion
Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. Begin by creating a form on your Gatsby site that you can receive submissions from. When creating the form, direct the Http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```

**Comment by sidharthachatterjee** [comment]
@mertcanyucel Can't seem to be able to push to your fork. There are some linting issues. Can you please run `yarn run format` and push up the changes so that linting checks on CI pass? 

We'll get this in then! Thanks!

**Comment by mertcanyucel** [comment]
@sidharthachatterjee I have done the formatting, Could you please check it and merge if it's ok?

**Comment by laurieontech** [comment]
Looks great! Thanks for the PR.

**Pull request merged**